### DISCUSSION TITLE
Hi, Impressed by your works. do you mind adding how to build guide?

### DISCUSSION THREAD

**Discussion by Turdari** [danielnilsson9/bbs-fw#219] [Q&A] [answered]
I'd like to go deep on your project but don't know how to, 
Or can you give me little tips?

**Comment by CitizenDev** [comment, marked as answer]
There is a guide under wiki
https://github.com/danielnilsson9/bbs-fw/wiki/Build-from-Source

**Reply by Turdari** [reply]
thanks, I'll read it!
